apps/meteor/ee/app/livechat-enterprise/server/hooks/afterOnHoldChatResumed.ts

```diff
@@ -1,20 +1,26 @@
+import type { IOmnichannelRoom } from '@rocket.chat/core-typings';
+
 import { callbacks } from '../../../../../lib/callbacks';
-import { LivechatEnterprise } from '../lib/LivechatEnterprise';
+import { AutoCloseOnHoldScheduler } from '../lib/AutoCloseOnHoldScheduler';
 import { cbLogger } from '../lib/logger';
 
-const handleAfterOnHoldChatResumed = async (room: any): Promise<void> => {
-	if (!room || !room._id || !room.onHold) {
-		cbLogger.debug('Skipping callback. No room provided or room is not on hold');
-		return;
+const handleAfterOnHoldChatResumed = async (room: Pick<IOmnichannelRoom, '_id'>): Promise<Pick<IOmnichannelRoom, '_id'>> => {
+	if (!room?._id) {
```

Comment:
shouldn't you add `!room.onHold` back?

Reply:
No, its not required now since this callback will only get called when a room is resumed from on-hold, since its `livechat:afterOnHoldChatResumed` :sweat_smile: